datalad/distributed/create_sibling_gitlab.py

```diff
@@ -119,6 +122,15 @@ class CreateSiblingGitlab(Interface):
         Project location/path used for a datasets at GitLab instance
         SITENAME (see --project). Configuring this is useful for deriving
         project paths for subdatasets, relative to superdataset.
+    "datalad.gitlab-default-projectname"
+        The hierarchy and collection layouts publish (sub)datasets as projects
```

Comment:
```suggestion
        The hierarchy and collection layouts publish (super)datasets as projects
```

I believe this is what the gitlab group taking the namespace slot of a (super)data contains as a project for that (super)dataset.